.idea/misc.xml

```diff
@@ -3,5 +3,5 @@
   <component name="JavaScriptSettings">
     <option name="languageLevel" value="ES6" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_14" default="false" project-jdk-name="Pipenv (timeatlas)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.7 (timeatlas-kzsAXcI_)" project-jdk-type="Python SDK" />
```

Comment:
Try to avoid making PR with IDE related files

Reply:
I'll update the .gitignore accordingly